test/solver_options.jl

```diff
@@ -0,0 +1,15 @@
+using OrdinaryDiffEq, StochasticDiffEq, Flux, DiffEqFlux, Test
+
+xs = Float32.(hcat([0.; 0.], [1.; 0.], [2.; 0.]))
+tspan = (0.0f0,25.0f0)
+dudt = Chain(Dense(2,50,tanh),Dense(50,2))
+
+@testset "only end" begin
```

Comment:
These tests need to be expanded to include other solver options. Especially the various ones which control the timepoints returned. I've noticed some combination of `save_*` break when trying to only return the final time step.

Reply:
interesting, yes please share

Reply:
these should all be fine nowadays.